include/swift/Frontend/FrontendOptions.h

```diff
@@ -283,6 +283,8 @@ class FrontendOptions {
   /// -dump-scope-maps.
   SmallVector<std::pair<unsigned, unsigned>, 2> DumpScopeMapLocations;
 
+  bool UseSharedResourceFolder = true;
```

Comment:
Doc comment please (its obvious in the context of the change, but less so when you just have a type to work with).